src/main/java/net/dv8tion/jda/api/events/message/package-info.java

```diff
@@ -21,5 +21,12 @@
  *
  * <p>These events combine all {@link net.dv8tion.jda.api.entities.MessageChannel MessageChannel}
  * messages but specifications can be found in subpackages.
+ *
+ * <h2>Requirements</h2>
+ *
+ * <p>Due to them being combinations, their requirements are a bit narrowed down.
```

Comment:
I honestly don't understand what this sentence means. How is it "narrowed down"?

Reply:
They require at least one of two intents, each with a specific characteristic. Since the events are more abstract (guild channels and/or private channels) the intents they need should be more specific to determine when they will fire. If you'd like for me to change it, what would you suggest

Reply:
Just say for guild messages you need the guild messages intent and for private messages the direct messages intent. It really isn't that complicated 🤔